pkgs/build-support/singularity-tools/default.nix

```diff
@@ -108,14 +116,19 @@ rec {
 
             # Build /bin and copy across closure
             mkdir -p bin ./${builtins.storeDir}
-            for f in $(cat $layerClosure) ; do
-              cp -ar $f ./$f
-            done
+            while IFS= read -r f; do
+              cp -r "$f" "./$f"
```

Comment:
Let's add a short comment that this used to be `-a`, then wait for ofborg-eval and merge

Reply:
On the second thought, `-a` is arbitrary and unnecessary, no comment needed